### PR TITLE
prMessage: recommend usage of nixpkgs-merge-bot

### PR DESCRIPTION
**Pull request by SigmaSquadron**
Nixpkgs now uses the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot) for maintainers who aren't committers, in order to combat Nixpkgs' "PR Crisis".

However, [some maintainers are unaware](https://github.com/NixOS/nixpkgs/pull/335431) that they can merge `nixpkgs-update` PRs using the bot, partly because the update message makes no mention of the bot.

This PR rectifies that by adding the following tip in the `maintainer pings` section:

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

I hereby license my contributions to this repository under:
Creative Commons Zero v1.0 Universal (SPDX Short Identifier: CC0-1.0)

### CODE REVIEW

File: src/Update.hs
Comment (zowoq) [inline src/Update.hs:611]: > after approving this update

Does this mean approving with github "review changes"?

Comment (SigmaSquadron) [reply src/Update.hs:611]: The bot itself doesn't really care if the PR is approved or not in GitHub, my reasoning was if a maintainer approves the changes made in the PR, then they probably tested it already, and if so, should merge.

Comment (SigmaSquadron) [reply src/Update.hs:611]: Would **after *testing* this update** be more appropriate?

Comment (zowoq) [inline src/Update.hs:611]:
```suggestion
       > As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this PR using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).
```

**Comment by SigmaSquadron** [comment]
I've also changed the wording of `PR` to `update` to keep the naming consistent with the rest of the message.

File: src/Update.hs
Comment (zowoq) [reply src/Update.hs]: Sorry, I missed this.

I'd prefer to be as concise as possible as PR message content is already rather long.

**Review by zowoq — APPROVED** [review]

**Comment by SigmaSquadron** [comment]
This appears to have been a success, and even when [it fails](https://github.com/NixOS/nixpkgs/pull/336262#issuecomment-2301388818), it encourages maintainers to port their packages to by-name!

**Pull request merged**
